features/providers/regressions/client_truelayer_partner_bank_statements_check_means_answer.feature

```diff
@@ -0,0 +1,84 @@
+    Then I should be on a page with title "Which of these payments does your client get?"
+    And the checkbox for Financial help from friends or family should be checked
+    And the checkbox for Maintenance payments from a former partner should be unchecked
+
+  @javascript
+  Scenario: Viewing client outgoings
+    When I click Check Your Answers Change link for "payments your client makes"
+    Then I should be on a page with title "Which of these payments does your client pay?"
+    And the checkbox for Childcare payments should be checked
+    And the checkbox for Maintenance payments to a former partner should be unchecked
+
+  @javascript
+  Scenario: Viewing partner payments
+    When I click Check Your Answers Change link for "payments the partner receives"
+    Then I should be on the "regular_incomes" page showing "Which of these payments does the partner get?"
+    And the checkbox for Financial help from friends or family should be unchecked
+    And the checkbox for Maintenance payments from a former partner should be checked
+
+  @javascript
+  Scenario: De-selecting and re-selecting client payments
+    When I click Check Your Answers Change link for "payments your client receives"
+    Then I should be on a page with title "Which of these payments does your client get?"
+    And the checkbox for Financial help from friends or family should be checked
+
+    When I select "My client does not get any of these payments"
+    And I click "Save and continue"
+    Then I should be on a page with title "Check your answers"
+    And the "Payments your client receives" section's questions and answers should match:
+      | question | answer |
+      | Benefits, charitable or government payments | None |
+      | Financial help from friends or family | None |
+      | Maintenance payments from a former partner | None |
+      | Income from a property or lodger | None |
+      | Pension | None |
+
+    When I click Check Your Answers Change link for "payments your client receives"
+    Then I should be on a page with title "Which of these payments does your client get?"
+
+    When I select "Financial help from friends or family"
+    And I click "Save and continue"
+    Then I should be on the "cash_income" page showing "Select payments your client receives in cash"
+
+    When I select "My client receives none of these payments in cash"
+    And I click "Save and continue"
+    Then I should be on the "income_summary" page showing "Sort your client's income into categories"
+
+    When I click link "View statements and add transactions"
+    Then I should be on the "incoming_transactions/friends_or_family" page showing "Select payments from friends or family"
+
+    When I select "babysitting"
+    And I click "Save and continue"
+    Then I should be on the "income_summary" page showing "Sort your client's income into categories"
+    And I should see "babysitting"
```

Comment:
might be worth adding a CYA result check step to see the updated amount, if there is a fixed monetary value associated with babysitting, or if the selector can take regexes?!

```
    And the "Payments your client receives" section's questions and answers should match:
      | question | answer |
      | Benefits, charitable or government payments | None |
      | Financial help from friends or family | "/£\d+" |
      | Maintenance payments from a former partner | None |
      | Income from a property or lodger | None |
      | Pension | None |
 ```

Reply:
That's not the same page, this would be on the income summary page:
![image](https://github.com/user-attachments/assets/e582a1d0-b0c9-4deb-a249-051c84cf441f)

We could do something like expect `Babysitting\t£44.00` but it would be a bit fragile